airflow/www/extensions/init_views.py

```diff
@@ -208,7 +208,11 @@ def _handle_method_not_allowed(ex):
             return views.method_not_allowed(ex)
 
     spec_dir = path.join(ROOT_APP_DIR, "api_connexion", "openapi")
-    connexion_app = App(__name__, specification_dir=spec_dir, skip_error_handlers=True)
+    if not conf.getboolean("webserver", "enable_swagger_ui"):
```

Comment:
If someone updates the code but doesn't update the conf file, this line will raise an error.

Reply:
This should be fixed in the latest commit. Thanks for noticing